ledfx/effects/audio.py

```diff
@@ -267,6 +267,7 @@ def open_audio_stream(device_idx):
                     ch = 2
 
             if hostapis[device["hostapi"]]["name"] == "WEB AUDIO":
```

Comment:
This is where I get worried, in that what web audio was this previously supporting, assuming the parameterization and differentiation of behavior should be driven from up here. Otherwise it feels like we are hijacking previous functionality. Admittedly I have no idea if the functionality is abandoned.

Reply:
Ah that's a good point. I don't think it is fully abandoned, so perhaps doing it this way may not be the best idea. I dont think these parameters accept anything directly from Yeon's websocket typescript code, but I may be completely wrong. Either way, a new Schema may be a good choice.